### PR TITLE
Not for review: https://github.com/flyteorg/flyte/issues/3729

### PR DESCRIPTION
**Pull request by katrogan**
TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by wild-endeavor — DISMISSED** [review]

**Review by hamersaw — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Review by hamersaw — APPROVED** [review]

**Review by EngHabu — COMMENTED** [review]
Do we need to cherry pick this to a previous release? Or release a patch for flyte 1.6?

**Comment by eapolinario** [comment]
For sure. We're going to have a patch release tomorrow morning.

**Comment by eapolinario** [comment]
Flyte 1.6.2 is [out](https://github.com/flyteorg/flyte/releases/tag/v1.6.2) and contains this fix.
